### PR TITLE
Use NcCheckboxRadioSwitch for QuestionMultiple

### PR DESCRIPTION
**Pull request by Chartman123**
This implements a TODO in `src\components\Questions\QuestionMultiple.vue` by using the vue component `NcCheckboxRadioSwitch`

Signed-off-by: Christian Hartmann <chris-hartmann@gmx.de>

### CODE REVIEW

**Comment by jotoeri** [comment]
Hmm:
- UI is still the old one, when editing the question
- I think we should make the styling more lightweight again. The Component-Styling is good for Settings or so, but for a form and esp. in combination with our current form styling, this is much too heavy.

**Comment by Chartman123** [comment]
> UI is still the old one, when editing the question

Yes, same as for the multiselect component... There is no direct way of adding new entries with the new component.

> make the styling more lightweight

You mean too much colors? I'm more a fan of using the same appearance througout the whole cloud and wouldn't change too much of the original styling of the component. 😉  I also thought about adjusting the styling of our "fake" checkboxes and radios in the edit mode...

**Comment by jotoeri** [comment]
>  I also thought about adjusting the styling of our "fake" checkboxes and radios in the edit mode...

Yeah, that's what i meant with my first point of the editing UI. That we should definitely do, to have the same styling as in non-editing view. The similar styling is also the reason why we created these fake-boxes at all. :wink: 

> same appearance througout the whole cloud 

Similar styling, yes. But we should also have in mind, that we need the end-user's focus on the form content, not on a fancy UI. The User should think about answering the questions instead of beeing deflected by coloured effects.
So - Yes, i think we'll need to go with NC towards the more blueish design, but i also think we need to reduce the weight to keep the focus on the actual content. In this case spontaneously, e.g. smaller borders and no hover-background. We'll probably also need to rework on the total design for that to have it commonly done.

**Comment by Chartman123** [comment]
What do you think about adjusting the style directly in the Vue component as you did for the NcListItem?

I also already thought about using the buttonVariant and buttonVariantGrouped = "vertical" to no longer have the radio/checkboxes. That way the design of these two elements would match the MultiSelect. But on the other hand, users can't see directly if it's a radio or checkbox field.

**Comment by Chartman123** [comment]
Ok, for now I've adjusted the editing view to match the default stlying of the component. 👍🏻

**Comment by Chartman123** [comment]
@jotoeri After today's meeting, should we leave it this way for the moment and merge it?

**Comment by jotoeri** [comment]
Can you please rebase it onto #1338 once merged and then go to align the Boxes and Text to make it pixel-perfect between edit/non-edit of a question (on create.vue). That also was one of the bigger things on the review.

File: src/components/Questions/QuestionMultiple.vue [hunk 1]
```diff
@@ -359,24 +343,23 @@ export default {
...
 		flex-shrink: 0;
 		display: inline-block;
-		height: 16px;
-		width: 16px !important;
+		height: 18px;
+		width: 18px !important;
 		vertical-align: middle;
-		margin: 0 14px 0px 0px;
-		border: 1px solid #878787;
+		margin: 7px 8px 0 1px;
```
[the remark this answers is not in the document]
Comment (Chartman123) [reply src/components/Questions/QuestionMultiple.vue:349]: Ok, strange... I adjusted it on FF until it stopped jumping.

[the remark this answers is not in the document]
Comment (jotoeri) [reply src/components/Questions/QuestionMultiple.vue:349]: Ok. So then we need to live with it for now. 🤷‍♂️

**Review by jotoeri — APPROVED** [review]
Mabe some more love for less jumping of the text would be good somewhen. But should suffice for now. 🙈

File: src/components/Questions/QuestionMultiple.vue (hunk 1 above)
[the remark this answers is not in the document]
Comment (Chartman123) [reply src/components/Questions/QuestionMultiple.vue:349]: I've just checked it with Chromium, too. No jumping there either

[the remark this answers is not in the document]
Comment (jotoeri) [reply src/components/Questions/QuestionMultiple.vue:349]: Hmm, probably its some question of screen resolution or so...

**Pull request merged**